tsconfig.json

```diff
@@ -30,7 +30,10 @@
       "@/types/*": ["./src/types/*"],
       "@/products/*": ["./src/products/*"],
       "@/playwright/*": ["./playwright/*"],
-      "react": ["./node_modules/@types/react"]
+      // Import the JS instead of the TS to avoid React 19 compatibility issues
+      "@department-of-veterans-affairs/component-library/dist/react-bindings": [
+        "./node_modules/@department-of-veterans-affairs/component-library/dist/react-bindings/index.js"
+      ]
```

Comment:
There's a whole long Slack thread for this. The TL;DR is that the component library adds their `.ts` files to `dist/`, so when we import from the library, the module resolution favors the `.ts` files and type checking is done on those library files. React 19 introduces some stricter static types (apparently) and type checking these files inside the component library resulted in some errors. So instead of making it our upstream dependency a blocker to migration, I decided to just import the `.js` files and let that library take care of its own types.

From what I can tell, there's no compatibility problem with React 19 and our component library other than the static types, but we'll want to do some solid QA.

:bat-signal: @laflannery 😇